src/pages/editor/PublishingStrategyOverview.tsx

```diff
@@ -0,0 +1,136 @@
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import RemoveCircleIcon from '@mui/icons-material/RemoveCircle';
+import { Box, Divider, Link, styled, Typography } from '@mui/material';
+import { Helmet } from 'react-helmet-async';
+import { Trans, useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { PageSpinner } from '../../components/PageSpinner';
+import { RootState } from '../../redux/store';
+import { CustomerRrsType } from '../../types/customerInstitution.types';
+import { StyledAccessRight, StyledAccessRightsContainer } from './PublishStrategySettings';
+
+const PublishStrategyContainer = styled('div')({
+  padding: '0.5rem',
+  borderRadius: '0.5rem',
+  backgroundColor: 'white',
+  boxShadow: '0px 3px 3px 0px rgba(0, 0, 0, 0.30)',
+});
+
+export const PublishingStrategyOverview = () => {
+  const { t } = useTranslation();
+  const customer = useSelector((store: RootState) => store.customer);
+
+  const coalitionSUrl = 'https://www.coalition-s.org/resources/rights-retention-strategy/';
```

Comment:
kan legges utenfor komponent, da den ikke trengs å lages på nytt for hver render